powerauth-restful-security-javaee/src/main/java/io/getlime/security/powerauth/rest/api/jaxrs/service/v2/SecureVaultService.java

```diff
@@ -134,7 +135,7 @@ public VaultUnlockResponse vaultUnlock(String signatureHeader,
         } catch (PowerAuthAuthenticationException ex) {
             throw ex;
         } catch (Exception ex) {
-            logger.warn("PowerAuth vault unlock failed", ex);
+            logger.warn("PowerAuth vault unlock failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.